test/test_pipeline/components/data_preprocessing/test_data_preprocessing_numerical.py

```diff
@@ -0,0 +1,60 @@
+import unittest
+import numpy as np
+from scipy import sparse
+
+from sklearn.utils.testing import assert_array_almost_equal
+
+from autosklearn.pipeline.components.data_preprocessing.data_preprocessing_numerical \
+    import NumericalPreprocessingPipeline
+
+
+class NumericalPreprocessingPipelineTest(unittest.TestCase):
+
+    def test_fit_transform(self):
+        X = np.array([
+            [3.14, 1.,     1.],
+            [3.14, 2., np.nan],
+            [3.14, 3.,     3.]])  # noqa : matrix legibility
+        # 1st column should be droped due to low variance
+        # The 2nd should be be standardized (default rescaling algorithm)
+        # The 3rd will get a value imputed by the mean (2.), therefore the transformation
+        # here will have the same effect as on the the 2nd column
+        sdev = (2 / 3) ** .5
```

Comment:
While it's the same, could you please use `np.sqrt`? It takes a bit less time to read the code.

Reply:
sure